### PR TITLE
mccs: restrain dune version

### PR DESCRIPTION
**Pull request by rjbou**
Update mccs 1.1+{5-9} packages to restrain dune version (cf https://github.com/ocaml/dune/issues/734#issuecomment-444177134)
Preferably not to merge before #13089 
cc @AltGr

### CODE REVIEW

File: packages/mccs/mccs.1.1+5/opam [hunk 1]
```diff
@@ -8,10 +8,10 @@ homepage: "http://www.i3s.unice.fr/~cpjm/misc/"
 bug-reports: "https://github.com/AltGr/ocaml-mccs/issues"
 license: "LGPL-2.1 with OCaml linking exception, BSD-3-clause, GPL-3.0"
 dev-repo: "git+https://github.com/AltGr/ocaml-mccs.git"
-build: ["jbuilder" "build" "-p" name]
+build: ["dune" "build" "-p" name]
 depends: [
   "ocaml"
-  "jbuilder" {build & >= "1.0+beta14"}
+  "dune" {build & (>= "1.0+beta14" | < "1.6.0" | > "1.6.1")}
```
Comment (kit-ty-kate) [inline packages/mccs/mccs.1.1+5/opam]: The `>= "1.0+beta14"` constrain is unnecessary (and probably wrong as `1.0 < 1.0+beta14`)

File: packages/mccs/mccs.1.1+6/opam [hunk 2]
```diff
@@ -8,10 +8,10 @@ homepage: "http://www.i3s.unice.fr/~cpjm/misc/"
 bug-reports: "https://github.com/AltGr/ocaml-mccs/issues"
 license: "LGPL-2.1 with OCaml linking exception, BSD-3-clause, GPL-3.0"
 dev-repo: "git+https://github.com/AltGr/ocaml-mccs.git"
-build: ["jbuilder" "build" "-p" name]
+build: ["dune" "build" "-p" name]
 depends: [
   "ocaml"
-  "jbuilder" {build & >= "1.0+beta19.1"}
+  "dune" {build & (>= "1.0+beta19.1" | < "1.6.0" | > "1.6.1")}
```
Comment (kit-ty-kate) [inline packages/mccs/mccs.1.1+6/opam]: same here

File: packages/mccs/mccs.1.1+5/opam (hunk 1 above)
Comment (rjbou) [reply packages/mccs/mccs.1.1+5/opam]: Updated!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**